worker-configuration.d.ts

```diff
@@ -10,5 +10,10 @@ export interface Env {
 	DATABASE_DO: DurableObjectNamespace<import('./src/index').DatabaseDO>;
 	SCHEDULER_DO: DurableObjectNamespace<import('./src/index').SchedulerDO>;
 	TOOLS_DO: DurableObjectNamespace<import('./src/index').ToolsDO>;
-	OPENAI_API_KEY: string;
+	IMAGES_DO: DurableObjectNamespace<import('./src/index').ImageGeneratorDO>;
+	METADATA_DO: DurableObjectNamespace<import('./src/index').TokenMetadataDO>;
+	CDN_DO: DurableObjectNamespace<import('./src/index').CdnDO>;
+	AIBTCDEV_SERVICES_BUCKET: R2Bucket;
+	AIBTCDEV_SERVICES_DB: D1Database;
+	OPENAI_API_KEY: process.env.OPENAI_API_KEY;
```

Comment:
This value should be string, you can use it through the worker env that's part of the request.

```suggestion
	OPENAI_API_KEY: string;
```